source/configure/calls-deployment.rst

```diff
@@ -0,0 +1,347 @@
+  net.core.optmem_max = 16777216
+
+Frequently asked questions
+--------------------------
+
+Is using UDP a requirement?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, UDP is the recommended protocol to serve real-time media as it allows for the lowest latency between peers. While theoretically possible to use TCP, it's not currently supported by the plugin. If using a UDP port is unfeasible, one possible solution would be to run calls through an external TURN server that listens on TCP and relays all media traffic between peers. However, this is a sub-optimal solution that should be avoided if possible as it will introduce extra latency along with added infrastructural cost.
+
+Do I need a TURN server?
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+TURN becomes necessary when you expect to have clients that are unable to connect through the configured UDP port. This can happen due to very restrictive firewalls that either block non standard ports even in the outgoing direction or don't allow the use of the UDP protocol altogether (e.g. some corporate firewalls). In such cases TURN is needed to allow connectivity.
+
+We officially support and recommend using `coturn <https://github.com/coturn/coturn>`_ for a stable and performance TURN service implementation.
+
+How will this work with an existing reverse proxy sitting in front of Mattermost?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Generally clients should connect directly to either Mattermost or, if deployed, the dedicated ``rtcd`` service through the configured UDP port . However, it's also possible to route the traffic through an existing load balancer as long as this has support for routing the UDP protocol (e.g. nginx). Of course this will require additional configuration and potential changes to how the plugin is run as it won't be possible to load balance the UDP flow across multiple instances like it happens for HTTP.
+
+Do calls require a dedicated server to work or can they run alongside Mattermost?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The plugin can function in different modes. By default calls are handled completely by the plugin which runs as part of Mattermost. It's also possible to use a dedicated service to offload the computational and bandwidth costs and scale further (Enterprise only).
+
+Can the traffic between Mattermost and ``rtcd``  be kept internal or should it be opened to the public?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When possible, it's recommended to keep communication between the Mattermost cluster and the dedicated ``rtcd`` service under the same private network as this can greatly simplify deployment and security. There's no requirement to expose ``rtcd``'s HTTP API to the public internet.
+
+Debugging
+---------
+
```

Comment:
As mentioned in https://github.com/mattermost/docs/pull/5838#issuecomment-1234562101 I was looking to port this whole section on debugging to [channels/make-calls.rst](https://github.com/mattermost/docs/pull/5838/files#diff-3896bf7e296e59501ca1aad56c5bb2f8969b317fc96e08dde75efc9216961ae1) since it's unrelated to deploying calls but more about actual use.

Reply:
Thanks, @streamer45 - sorry about that! 🤦‍♀️